src/image.rs

```diff
@@ -27,43 +27,94 @@ impl Format {
     }
 }
 
+/// Defines the desired quality for sampling an [image](Image).
+#[derive(Copy, Clone, PartialEq, Eq, Default, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum ImageQuality {
+    /// Lowest quality with best performance characteristics.
+    ///
+    /// This is typically nearest neighbor sampling.
```

Comment:
I'd quite like to see `NearestNeighbour` available as an explicit option, because for certain applications it is something you want to force.